bluebottle/time_based/models.py

```diff
@@ -1066,6 +1129,38 @@ class JSONAPIMeta(object):
         resource_name = 'contributors/time-based/deadline-participants'
 
 
+class PeriodicSlot(TriggerMixin, models.Model):
```

Comment:
We don't want to make slots polymorphi? Is there too little in common between the different slot types?

Reply:
I do not want to change the date slot model now. If we move date activities to registrations, then I'd make them a subclass.
And the to-be-decided activity will share a parent with this model.